prow/scripts/cluster-integration/kyma-aks-long-lasting.sh

```diff
@@ -53,7 +53,7 @@ readonly CURRENT_TIMESTAMP=$(date +%Y%m%d)
 export CLUSTER_NAME="${STANDARIZED_NAME}"
 export CLUSTER_SIZE="Standard_D4_v3"
 # set cluster version as MAJOR.MINOR without PATCH part (e.g. 1.10, 1.11)
-export DEFAULT_CLUSTER_VERSION="1.14"
+export DEFAULT_CLUSTER_VERSION="1.16"
```

Comment:
K8s 1.16 is in preview in AKS, so it is not stable yet...

Reply:
Reverted